README.md

```diff
@@ -251,3 +251,10 @@ If you must preserve permissions, you can `tar` all of your files together befor
     name: my-artifact
     path: my_files.tar
 ```
+
+# Recommended Permissions
+
+The `actions/download-artifact` workflow relies on an internal authentication pattern and does not use the GITHUB_TOKEN, to reduce risk of over-privileged token, jobs that use `actions/download-artifact` should set permissions to none:
```

Comment:
Similar to https://github.com/actions/cache/pull/1536#discussion_r1951686131, I'm worried this language will mislead users to setting up a workflow with no permissions that would likely fail.